docs/pages/components/button.md

```diff
@@ -8,116 +8,67 @@ permalink: components/button.html
 folder: components
 ---
 
-Buttons allow users to perform actions.
-{: .docs-intro}
-The priority of buttons within a page should be considered. For instance, only use the main button once within a page or dialog.
-Color is also important. For instance, the most important button has a blue background where as a red button should only be used if the action it performs is potentially destructive.
-
-<br>
-
-## Button Options
-There are three emphasis styles used to indicate the importance of the button on the page.
-- **Emphasized Button**: There should only be one highlighted button on the page. This is the primary call to action.
-- **Regular Button**: The default button style and the most common button. There may be more than one on a page.
-- **Light Button**: This is the lowest priority button and most often used with page content like appearing in a table or list. There may be more than one on the page.
+Buttons allow users to perform actions. All the buttons require base class `fd-button` and additional modifier for all the flavours.
 
 {% capture button %}
-<button class="fd-button fd-button--emphasized">Emphasized Button</button>
-<button class="fd-button">Regular Button</button>
-<button class="fd-button fd-button--light">Light Button</button>
+<button class="fd-button">Create</button>
+<button class="fd-button sap-icon--cart"></button>
+<button class="fd-button fd-button--emphasized ">Save</button>
 {% endcapture %}
 {% include display-component.html component=button %}
 
-## Button Types
+{: .docs-intro}
+There are 4 button types:
+
+- **Simple button** for one action.
+- **Toggle button** for switching between different states.
+- **Segmented button** for a group of options.
+- **Menu button** with a group of actions.
 
-There are five button types that can be combined with the emphasis styles.
-- **Action Button**: The default button
-- **Standard Button**: Neutral or informative color
-- **Positive Button**: Used for positive actions such as approved, ok, yes.
-- **Medium Button**: Used for warnings or inside message strips
-- **Negative Button**:Used for negative actions such as decline, cancel, no.
+<br>
 
+## Design Types
+
+- **Default Button**
+- **Emphasized Button** Used for primary action
+- **Ghost Button** It is used for secondary actions or primary button in case there is already a page primary button
+- **Standard Button** Neutral or informative (secondary) actions
+- **Positive Button** Used for positive (secondary) actions
+- **Negative Button** Used for negative (secondary) actions
+- **Attention Button**
+- **Transparent Button** Used for secondary or negative path actions
 
```

Comment:
What is the diff between Default and Standard?

Reply:
I removed the standard